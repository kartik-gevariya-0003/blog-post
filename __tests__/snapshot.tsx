import { act, render } from '@testing-library/react'
import Home from '@/pages/index'
import { QueryClient, QueryClientProvider } from 'react-query';

it('renders homepage unchanged', async () => {
  const queryClient = new QueryClient();
  // @ts-ignore
  const {container} = await act(async () => render(
    <QueryClientProvider client={queryClient}>
      <Home/>
    </QueryClientProvider>
  ));

  expect(container).toMatchSnapshot()
})
