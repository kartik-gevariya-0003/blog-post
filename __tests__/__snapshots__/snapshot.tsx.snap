// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h1
        class="title"
        role="heading"
      >
        Welcome to BlogPost!
      </h1>
      <div
        class="justify-content-center row"
      >
        0
        <div
          class="mt-3 justify-content-center row"
        >
          <div
            class="col"
          >
            <nav
              aria-label="Page navigation example"
            >
              <ul
                class="pagination justify-content-center"
              >
                <li
                  class="page-item disabled"
                >
                  <a
                    aria-label="Previous"
                    class="page-link"
                  >
                    <span
                      aria-hidden="true"
                    >
                      «
                    </span>
                  </a>
                </li>
                <li
                  class="page-item"
                >
                  <a
                    aria-label="Next"
                    aria-labelledby="Next"
                    class="page-link"
                  >
                    <span
                      aria-hidden="true"
                    >
                      »
                    </span>
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer"
      role="footer"
    >
      Developed by Kartik Gevariya
    </footer>
  </div>
</div>
`;
